### PR TITLE
Second Branch I created

### PR DESCRIPTION
**Pull request by Louisaflor**
Was able to create the functionality for the characteristics scale. @noattongi @xiaxia330 @akblal Please review

### CODE REVIEW

**Comment by Louisaflor** [comment]
I npm installed cloudinary so just wanted to give you guys a heads up when you pull down my work!

**Review by xiaxia330 — APPROVED** [review]
NICE

File: client/src/Q&A/Q&A.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, {useEffect, useState} from 'react';
 import axios from 'axios';
 import Search from './Search.jsx';
 import QuestionList from './QuestionList.jsx';
-import AddQuestionButton from './AddQuestionButton.jsx';
+// import AddQuestionButton from './AddQuestionButton.jsx';
```
Comment (noattongi) [inline client/src/Q&A/Q&A.jsx:5]: Sorry if this gave you errors. I thought I had removed this.

**Review by noattongi — APPROVED** [review]
Nice job.

**Pull request merged**